tests/Crypto/tst_cryptorequests/tst_cryptorequests.cpp

```diff
@@ -3361,17 +3361,19 @@ void tst_cryptorequests::importKey_data()
             << 0
             << Key::OriginUnknown
             << CryptoManager::AlgorithmUnknown;
-    QTest::newRow("Private RSA 2048 - passphrase, canceled")
-            << test_key_rsa_2048_sailfish_in
-            << createPrivateKey(keyIdentifier)
-            << promptToCancel
-            << Result::Failed
-            << Result::CryptoPluginKeyImportError
-            << QByteArray()
-            << QByteArray()
-            << 0
-            << Key::OriginUnknown
-            << CryptoManager::AlgorithmUnknown;
+
+    // this test should be enabled for manual testing only, user has to cancel the dialog.
```

Comment:
Is this added intentionally here? Looks like it's not related to the current topic.

Reply:
Intentional, I noticed the discrepancy when I updated this test to use the proper plugin.  Figured it is better to ensure that all tests pass as a result of the commit, rather than splitting the plugin mapping and unit test fixing over two separate commits.